audit/src/main/java/com/ritense/audit/service/impl/AuditRetentionServiceImpl.java

```diff
@@ -33,6 +34,9 @@ public AuditRetentionServiceImpl(AuditService auditService, long retention) {
 
     @Override
     @Scheduled(cron = "${scheduling.job.cron.cleanupAuditEvents:-}")
+    @SchedulerLock(
+        name = "AuditRetentionService_cleanup", lockAtLeastFor = "PT4S", lockAtMostFor = "PT6S"
```

Comment:
'lockAtMostFor' 6 seconds feels a bit short. I would prefer it to be at least 1 hour.